packages/@textlint/kernel/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "newLine": "LF",
-    "outDir": "./es_modules/",
+    "outDir": "./lib/",
     "target": "esnext",
```

Comment:
Ah, Neet to change `"target": "esnext",` to `"target": "es5".

Reply:
Thank you for your advice.
I'd updated to `es5`. Wait for CI pass...